doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,386 @@
+
+      .. parsed-literal::
+         :class: highlight
+
+         nrfutil toolchain-manager install --bundle-id *custom_bundle_ID*
+
+      To read more about these commands, use the ``nrfutil toolchain-manager --help`` command.
+
+.. _cloning_the_repositories_win:
+.. _cloning_the_repositories:
+
+.. rst-class:: numbered-step
+
+Get the |NCS| code
+******************
+
+Every |NCS| release consists of a combination of Git repositories at different versions and revisions.
+The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, `sdk-nrf`_.
+In a nutshell, you can work with the following versions of the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - |NCS| version
+     - Required identifier of the revision
+     - Where to find the identifier
+   * - Specific release (recommended)
+     - Release tag (for example, |release_tt|)
+     - :ref:`Release_notes` of the release
+   * - :ref:`Development tag <dm-revisions>`
+     - Development tag (for example, ``v1.9.2-dev1``)
+     - :ref:`Changelog <release_notes>` of the tag
+   * - Branch
+     - Branch name (for example, ``main``)
+     - `sdk-nrf`_ repository
+
+.. note::
+   Unless you are familiar with the :ref:`development process <dev-model>`, you should always work with a specific release of the |NCS|.
+
+For more information about the repository and development model, see the :ref:`dm_code_base` page.
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      To clone the |NCS| code, complete the following steps:
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage SDKs`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install SDK`.
+         The list of available SDK versions appears in the |VSC|'s quick pick.
+      #. Select the SDK version to install.
+         In most cases, you want the SDK version that matches your toolchain version.
+         The SDK installation starts and it can take several minutes.
+
+   .. group-tab:: Command line
+
+      To clone the repositories, complete the following steps:
+
+      1. Open the directory :file:`ncs`.
+         This is one level up from the location where you installed the toolchain.
+         This directory will hold all |NCS| repositories.
+
+      #. Determine the identifier of the revision of the |NCS| you want to work with.
+         See the table below for more information.
+
+      #. On the command line, go to the :file:`ncs` directory (``cd ncs``) and initialize west with the revision of the |NCS| that you want to check out, replacing *nRFConnectSDK_revision* with the identifier:
+
+         .. parsed-literal::
+            :class: highlight
```

Comment:
isn't this just a full duplicate of the same information as the other commit has in installing.rst?

I see the file has moved, there should probably be a file redirect added from old to new URL

Reply:
Apologies for confusion, I've merged the two commits now.